winui/Masked-TextBox/Mask-Types.md

```diff
@@ -7,4 +7,192 @@ control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Mask types in WinUI SfMaskedTextBox
```

Comment:
@Vignesh-SF3580 Change the description.

Learn more about the various mask types available in the MaskedTextBox (SfMaskedTextBox) control, including Simple and RegEx, and how to use them effectively.

Reply:
Changed the description.